lib/api/models/test_types_model.dart

```diff
@@ -0,0 +1,34 @@
+class TestList {
```

Comment:
What's this for?

Reply:
Tested out a pokemon type model to use for pokemon tile.